### PR TITLE
[1.2.0] Update the changelog

### LINKED ISSUE #28: [1.2.0] Update the changelog
**Description:**
Update the changelog.

### PR DESCRIPTION
**Pull request by ThilinaManamgoda**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,14 @@ All notable changes to this project `1.2.x` per each release will be documented
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [v1.2.0.3] - 2020-09-22
+
+### Added
+- Added support for synapse testing mode
```
Comment (chirangaalwis) [inline CHANGELOG.md:9]: @ThilinaManamgoda shall we please add the issue link?

Also, when released please add the tested environments to the release notes. Please check the sample [change log](https://github.com/wso2/kubernetes-apim/blob/v3.2.0.2/CHANGELOG.md) and [release note](https://github.com/wso2/kubernetes-apim/releases/tag/v3.2.0.1) from here.

**Review by chirangaalwis — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,17 @@ All notable changes to this project `1.2.x` per each release will be documented
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [v1.2.0.3] - 2020-09-22
+
+###Environments
+- Successful evaluation of Micro Integrator Helm chart in Google Kubernetes Engine (GKE)
```
Comment (chirangaalwis) [inline CHANGELOG.md:9]: @ThilinaManamgoda let's add the GKE version as well.

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,4 +31,4 @@ For detailed information on the tasks carried out during this release, please se
 ](https://github.com/wso2/kubernetes-mi/milestone/1)
 
 
-[v1.2.0.2]: https://github.com/wso2/kubernetes-mi/compare/v1.2.0.1...v1.2.0.2
+[v1.2.0.3]: https://github.com/wso2/kubernetes-mi/compare/v1.2.0.2...v1.2.0.3
```
Comment (chirangaalwis) [inline CHANGELOG.md:34]: @ThilinaManamgoda let's maintain the commit diffs between other releases also.

e.g.

```
[v1.2.0.3]: https://github.com/wso2/kubernetes-mi/compare/v1.2.0.2...v1.2.0.3
[v1.2.0.2]: https://github.com/wso2/kubernetes-mi/compare/v1.2.0.1...v1.2.0.2
[v1.2.0.1]: https://github.com/wso2/kubernetes-mi/compare/<some-version>...v1.2.0.1
```

**Review by chirangaalwis — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (ThilinaManamgoda) [reply CHANGELOG.md:34]: Done

**Review by chirangaalwis — APPROVED** [review]

**Pull request merged**